.github/workflows/runtests.yml

```diff
@@ -22,7 +22,7 @@ jobs:
         with:
           python-version: '3.10'
       - run: python -m pip install --upgrade pip
-      - run: pip install mpmath flake8
+      - run: pip install mpmath flake8 flake8-comprehensions
```

Comment:
Does installing this cause it to be run automatically by flake8?

Reply:
@oscarbenjamin No, the rules have to be manually selected in the setup.cfg

Reply:
Is it because you've added the C code in setup.cfg?

https://github.com/sympy/sympy/blob/7ba640371de813a3c59b46d862bb019d21ac78ee/setup.cfg#L5-L15

Reply:
Yes, all the error codes begin with C4: https://pypi.org/project/flake8-comprehensions/

Reply:
So if flake8-comprehensions is installed and the C code is added in setup.cfg then that causes these checks to run?

I just want to be clear so that anyone looking at this in future can understand.

Also you have added other files to the exclude list here because they are automatically generated and so probably should not be inspected by these tools.

Reply:
Correct, if the plugin is installed it will add the C4 codes to flake8, which can be selected like other flake8 rules.

Yeah, I added the other generated files to the list of ignores since flake8 shouldn't complain about them.

Reply:
I originally didn't bother enabling C417 because it didn't have autofixits, but I've went and manually fixed it so all the comprehension checks should be enabled.